packages/render-fidelity-tools/test/config.json

```diff
@@ -385,6 +385,18 @@
         "dspbr-pt"
       ]
     },
+    {
+      "name": "khronos-Fox",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/Fox/glTF/Fox.gltf",
+      "orbit": {
```

Comment:
Let's adjust the target to center the model (Y is up, so that may be all you need to change). And probably reduce the height, since it's a wide model.

Reply:
How is the latest fox?

Reply:
Looks like you could zoom in by about a factor of 2. The vast majority of the pixels are blank.